packages/apps/docs/src/docs/build/pact/beginner/language-basics.md

```diff
@@ -1,205 +1,196 @@
 ---
-title: Explore Pact Language Basics
-description: Kadena makes blockchain work for everyone.
+title: Explore basic functions
+description: Start learning the Pact programming language with an introduction to built-in functions and writing your own simple functions.
 menu: Smart contracts
-label: Explore language basics
+label: Explore basic functions
 order: 4
 layout: full
 tags: ['pact', 'language basics']
 ---
 
-# Explore Pact language basics
+# Explore basic functions
 
-In [Set up a local development network](/build/pact/dev-network), you learned how to set up a local development environment and connect to it using Chainweaver. As part of the introduction to the development environment, you saw how to navigate smart contracts and the basics of using Chainweaver developer tools, including the interactive Pact REPL.
+In [Set up a local development network](/build/pact/dev-network), you learned how to set up a local development environment and connect to it using Chainweaver. 
+Chainweaver provides a Module Explorer as part of its development environment.
+The Module Explorer enables you to navigate smart contract modules—that is, `.pact` files—review module code, and view or call module functions. 
+Chainweaver also includes an editor for working directly with the code and the Pact read-evaluate-print-loop (REPL) interactive interpreter for testing your code as you go.
+In most cases, you can use the account and contract management features in Chainweaver in combination with an integrated development environment (IDE), like Visual Studio Code, or another code editor to provide end-to-end development environment. 
 
-In this tutorial, you'll learn some more fundamental concepts for working with Pact as a smart contract language and explore some of the most important built-in functions that Pact provides.
+In this tutorial, you'll learn the basics for working with the Pact programming language and the built-in functions that Pact provides using the Pact REPL interpreter and Visual Studio Code.
+If you use Chainweaver or another IDE for your code editor, you'll need to adjust some steps to suit your environment. 
 
-**Topics covered in this tutorial**
+## Basic operations and data types
 
-- Pact Language Reference
-- Pact Syntax
-- Basic Commands
-- Built in Functions
-- Create Functions
+Pact, like most programming languages, supports the data types that you would expect.
+For example, Pact allows you define the following types of data:
 
-:::note Key Takeaway
+| Data type | Description | Examples
+| :--------- | :----------- | :-------
+| Integer | Any whole number value—positive or negative—that doesn't include a decimal.| 1, 2, 3, -19
+| Decimal | Any number value that includes a decimal. Decimal values can have a potentially unlimited precision. | 1.0, 23.5, 3.14159265359
+| String | Any text within quotes. You can represent strings using double quotes or, in some cases, using a single quote. | “Hello”, "Welcome to the show", 'hello
```

Comment:
Do we support single-quotes for string literals? REPL rejects this for me (using pact-5, and legacy pact)

Reply:
Oops, I see we're talking about lisp symbols, not surrounding a string with single quotes.

The examples are good `"hi"` and `'hi` are both valid strings, although the prose is a little confusing.

Reply:
Yeah, this confused me some too. I think I'll revise this.